README.rdoc

```diff
@@ -173,6 +173,28 @@ Or how about persisting the session...
       end
   end
 
+
+== CSRF Protection
+
+Because Authlogic introduces its own methods for storing user sessions, the CSRF (Cross Site Request Forgery) protection that is built into Rails will not work out of the box.
```

Comment:
@lukeasrodgers wondering if we should emphasize here that it's a user-defined session (eg `def current_user_session`) so there's no way for Authlogic to know what to reset?